src/modules/leaderboard.ts

```diff
@@ -0,0 +1,118 @@
+import { Sequelize, Op } from "sequelize";
+
+import db from "src/db";
+import logger from "src/utils/logger-handler";
+
+async function updateLeaderboardRow(address: string, property: string, value: number) {
+  const userLeaderboard = await db.leaderboard.findOne({
+    where: { address }
+  });
+
+  if(userLeaderboard) {
+    userLeaderboard[property] = value;
+
+    await userLeaderboard.save();
+  } else
+    await db.leaderboard.create({
+      address,
+      [property]: value,
```

Comment:
same here, we can't _simply add new props_ as the model will complain if we try to be resourceful with it, we need to make sure that `property` is an allowed property inside the model before using it.